src/gui/aboutdialog.cpp

```diff
@@ -60,7 +60,7 @@ AboutDialog::AboutDialog(QWidget *parent)
         "</table>"
         "</p>")
         .arg(tr("An advanced BitTorrent client programmed in C++, based on Qt toolkit and libtorrent-rasterbar.")
-            , tr("Copyright %1 2006-2020 The qBittorrent project").arg(QString::fromUtf8(C_COPYRIGHT))
+            , tr("Copyright %1 2006-2021 The qBittorrent project").arg(QString::fromUtf8(C_COPYRIGHT))
```

Comment:
Guys, does the problem of having UTF-8 characters directly in strings still exist?

Reply:
IIRC it was due to something about MSVC that @Chocobo1 had encountered in the past. I didn't have any problem.

Reply:
>Guys, does the problem of having UTF-8 characters directly in strings still exist?

Yes and MS still let it exist.
If this is a nuisance, we can turn off that warning flag.

Reply:
>If this is a nuisance, we can turn off that warning flag.

Does it bring any real trouble beyond a warning? If not, I'd rather get rid of this workaround by using character substitution from a separate file.